client/src/components/Overview/ProductInfo.jsx

```diff
@@ -1,25 +1,82 @@
-import React from 'react';
-
-function ProductInfo({ description, styles }) {
-  // let price;
-  // if (styles.results !== undefined) {
-  //  if(styles.result[0].sale_price !== null){
-  //   price = <p>{styles.result[0].original_price}  {description.default_price}</p>
-  // } else {
-  //   price =<p>{description.default_price}</p>
-  // }
-  // }
-  return (
-    <div>
-      <p>stars and review</p>
-      <p> {description.category}</p>
-      <p>{description.name}</p>
-      <div> {description.default_price}</div>
-      <p>
-        Social Media Icons
-      </p>
-    </div>
-  );
+/* eslint-disable arrow-body-style */
+/* eslint-disable react/destructuring-assignment */
+import React, {
+// useState,
+// useEffect,
+// useContext
+} from 'react';
+import axios from 'axios';
+import { FaFacebook, FaPinterest } from 'react-icons/fa';
+// eslint-disable-next-line import/no-unresolved
+import { CgTwitter } from 'react-icons/Cg';
+import PropTypes from 'prop-types';
+
+class ProductInfo extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      description: [],
+    };
+  }
+
+  componentDidMount() {
+    // eslint-disable-next-line react/destructuring-assignment
```

Comment:
This linter disable is redundant, if you are including the one on line 2 for the whole file